CAPE/CAPE/run_cape_offline.cpp

```diff
@@ -103,12 +103,12 @@ int main(int argc, char ** argv){
         PATCH_SIZE = atoi(argv[1]);
         sequence = argv[2];
     }else{
-        PATCH_SIZE = 20;
-        sequence = "cy_0";
+        PATCH_SIZE = 16;
```

Comment:
It is the default parameters of data that be tested. If you have data then you should put it in one folder and this folder should be located in `input` directory. I added these changes that just to put data in `input` directory without gathering them in one folder again